src/odil/message/NCreateResponse.h

```diff
@@ -0,0 +1,60 @@
+/*************************************************************************
+ * odil - Copyright (C) Universite de Strasbourg
+ * Distributed under the terms of the CeCILL-B license, as published by
+ * the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+ * http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+ * for details.
+ ************************************************************************/
+ 
+#ifndef __NCREATERESPONSE_H__
+#define __NCREATERESPONSE_H__
```

Comment:
Please switch to the uuid-based include guards for coherence with the rest of the code.